src/components/manage/UniversalLink/UniversalLink.jsx

```diff
@@ -65,13 +65,25 @@ const UniversalLink = ({
     )?.length > 0;
   const isExternal = !isInternalURL(url) || isBlacklisted;
   const isDownload = (!isExternal && url.includes('@@download')) || download;
+  const isMail = URLUtils.isMail(url) || URLUtils.isMail('mailto:' + url);
+  if (isMail) {
+    url = URLUtils.normaliseMail(url);
+  }
+
+  const isTelephone =
```

Comment:
I think this code should be placed inside helpers/Url.js, as it's also used by the Link plugin for draft. https://github.com/plone/volto/blob/2b80b6c3f9811c5fbe069ee25236d41f762a8191/src/helpers/Url/Url.js#L231